gulp/tasks/styles.js

```diff
@@ -2,16 +2,17 @@ import gulp from 'gulp';
 import plumber from 'gulp-plumber';
 import sass from 'gulp-sass';
 import sourcemaps from 'gulp-sourcemaps';
-import uncss from 'gulp-uncss';
-import vendorPrefix from 'gulp-autoprefixer';
-import cleanCss from 'gulp-clean-css';
+import uncss from 'uncss';
+import cssnano from 'cssnano';
+import autoprefixer from 'autoprefixer';
+import postcss from 'gulp-postcss';
 import rename from 'gulp-rename';
 
 import {settings, useHandlebars} from '../config';
 import {isProdBuild} from '../commandLineArgs';
 import onError from '../onError';
 
-const htmlSources = useHandlebars ? './.metalsmith-build/*.html' : settings.sources.html;
+const htmlSources = useHandlebars ? './dist/**/*.html' : settings.sources.html;
```

Comment:
We could simplify this and take `dist`in every case no matter of html is generated from handlebars templates.
What do you think?